packages/make-styles/src/runtime/compileStaticCSS.ts

```diff
@@ -0,0 +1,18 @@
+import { compile, middleware, serialize, stringify, prefixer, rulesheet } from 'stylis';
+import { MakeStyles } from '../types';
+import { cssifyObject } from './utils/cssifyObject';
+
+export function compileStaticCSS(property: string, value: MakeStyles): string {
+  const cssRule = `${property} {${cssifyObject(value)}}`;
+  return compileCSSRule(cssRule);
+}
+
+export function compileCSSRule(cssRule: string): string {
+  return serialize(compile(cssRule), middleware([prefixer, stringify]));
+}
+
+export function compileCSSRules(cssRules: string): string[] {
+  const rules: string[] = [];
+  serialize(compile(cssRules), middleware([prefixer, stringify, rulesheet(rule => rules.push(rule))]));
```

Comment:
```suggestion

  // 💡 we are using ".insertRule()" API for DOM operations, it does not support insertion of multiple CSS rules in a single call
  //     "rulesheet" plugin extracts individual rules to be used with this API
  serialize(compile(cssRules), middleware([prefixer, stringify, rulesheet(rule => rules.push(rule))]));
```

Reply:
nit: I also suggest to move out `compileCSSRule` and `compileCSSRules` to a separate file